cdhweb/people/models.py

```diff
@@ -122,15 +122,14 @@ class ProfileQuerySet(PublishedQuerySetMixin):
 
     #: position titles that indicate a person is a postdoc
     postdoc_title = 'Postdoctoral Fellow'
-    #: variant postdoc title for Princeton PGRA
-    postgrad_title = 'Postgraduate Research Associate'
 
     #: position titles that indicate a person is a project director
     director_roles = ['Project Director', 'Co-PI: Research Lead']
 
     #: position titles that indicate a staff person is a student
     student_titles = ['Graduate Fellow', 'Graduate Assistant',
```

Comment:
For a future enhancement, should we put a student flag on the position model? To make it configurable instead of hard-coded here.  (What about UAFs? Do we need that title here too?)

Reply:
I like that idea; maybe we can move away from using PU status and just determine student/staff listings based on a flag set on the position title. I think UAFs will follow the pattern of already having their status set to Student, so we maybe don't need that here yet, but it'd be easy to add.